dockstore-webservice/src/main/java/io/dockstore/webservice/core/PrivacyPolicyVersion.java

```diff
@@ -0,0 +1,25 @@
+/*
+ *    Copyright 2019 OICR
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+package io.dockstore.webservice.core;
+
+import io.swagger.annotations.ApiModel;
+
+@ApiModel(description = "Enumerates which version, if any, of the privacy policy a user has accepted ")
+public enum PrivacyPolicyVersion {
```

Comment:
I wonder if this and TOSVersion should instead be plain strings, coming from dockstore.yml.

Let's say we need to do an "emergency" update, e.g., some country changes a law that forces us to update the TOS and/or the privacy policy. The way you have it, we will have to do a new web service build in addition to a new UI build (where the PDF's are committed).

On the other hand, it may never occur that we need an emergency update, and it's easier to deal with enums the way you have it.

Reply:
It's a good question. I think we decided in a storytime that we would aim to not make privacy policies/TOS changes more frequent than releases. But seeing it coded up and enforced gives me pause.